assets/js/components/DashboardViewIndicator.js

```diff
@@ -35,8 +36,15 @@ const DashboardViewIndicator = () => {
 	const isGA4DashboardView = useSelect( ( select ) =>
 		select( MODULES_ANALYTICS ).isGA4DashboardView()
 	);
```

Comment:
Somewhat unrelated to this fix, but this selector will raise a console error on the view-only dashboard because the setting won't be preloaded and the resulting request for settings will fail.

Reply:
For now, let's skip rendering this in a view-only context and we'll address this in a follow-up.

Reply:
Nevermind – **this point is null and this is not an issue for view-only users** 😄 

See https://github.com/google/site-kit-wp/pull/6872/files#diff-fd0d1b4a1cb0416c9df3564b6b7d69870d87563b32b6a187084c8c73f4a141c3